### PR TITLE
Add a blog post about configuring SSL using certificate revocation lists

### PR DESCRIPTION
**Pull request by SoniaZaldana**
Blog post providing overview on how to configure certificate revocation lists: https://issues.redhat.com/browse/ELY-2033

Has a missing link once the corresponding quickstart gets merged: https://github.com/wildfly-security-incubator/elytron-examples/pull/42

### CODE REVIEW

**Comment by fjuma** [comment]
@SoniaZaldana When you get a chance, would be good to update the date in the file name and in the post itself. Thanks!

File: _posts/2020-12-09-ssl-certificate-revocation-list.adoc [hunk 1]
```diff
@@ -0,0 +1,454 @@
+---
+layout: post
+title: 'SSL Certificate Revocation Lists with WildFly'
+date: 2020-12-09
+tags: elytron ssl
+synopsis: An overview of the certificate revocation lists support included in WildFly 22.
```
Comment (fjuma) [inline _posts/2020-12-09-ssl-certificate-revocation-list.adoc:6]: WildFly 24

File: _posts/2020-12-09-ssl-certificate-revocation-list.adoc [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+
+== An Overview of Certificate Revocation Lists
+
+Certificate Revocation Lists (CRLs) contain a list of certificates that have been revoked
+by the issuing Certificate Authority before their expected expiration date, and therefore should no longer
+be trusted. Elytron previously supported configuring only one Certificate Revocation List.
+
+It is now possible to configure multiple CRL files, which is useful when several certificate authorities are used.
+This blog post assumes previous knowledge on how to configure SSL in Elytron. For more information on how to configure SSL,
+refer to the elytron https://docs.wildfly.org/21/WildFly_Elytron_Security.html#configure-ssltls[documentation.]
```
Comment (fjuma) [inline _posts/2020-12-09-ssl-certificate-revocation-list.adoc:24]: We can update the docs link to 24

Comment (SoniaZaldana) [reply _posts/2020-12-09-ssl-certificate-revocation-list.adoc:24]: To ``https://docs.wildfly.org/24/WildFly_Elytron_Security.html#configure-ssltls``?

Comment (fjuma) [reply _posts/2020-12-09-ssl-certificate-revocation-list.adoc:24]: Yes. Thanks!

File: _posts/2020-12-09-ssl-certificate-revocation-list.adoc (hunk 1 above)
Comment (SoniaZaldana) [reply _posts/2020-12-09-ssl-certificate-revocation-list.adoc]: Updated

**Review by fjuma — APPROVED** [review]

File: _posts/2020-12-09-ssl-certificate-revocation-list.adoc (hunk 2 above)
Comment (SoniaZaldana) [reply _posts/2020-12-09-ssl-certificate-revocation-list.adoc]: Updated

**Comment by SoniaZaldana** [comment]
> @darranl @fjuma This one has a missing link to the WildFly documentation on how to configure multiple CRLs in two way SSL. I think I need to wait until that documentation goes live.

Farah mentioned that it looks like there isn't an anchor that can be used to access it, so we can use the same doc link. I have now updated the doc link and this post should be good to go.

**Pull request merged**